src/compiler/compile/render_dom/Renderer.ts

```diff
@@ -199,26 +204,29 @@ export default class Renderer {
 			? x`$$self.$$.dirty`
 			: x`#dirty`) as Identifier | MemberExpression;
 
-		const get_bitmask = () => names.reduce((bitmask, name) => {
-			const member = renderer.context_lookup.get(name);
-
-			if (!member) return bitmask;
+		let bitmask;
+		const get_bitmask = () => {
+			const bitmask: BitMasks = [];
+			names.forEach((name) => {
+				const member = renderer.context_lookup.get(name);
 
-			if (member.index.value === -1) {
-				throw new Error(`unset index`);
-			}
+				if (!member) return bitmask;
```

Comment:
Returning `bitmask` doesn't actually do anything anymore, right? We should just be `return`ing here, because this is a `forEach` now (and the `return bitmask;` at the end should be completely removed I think).

Reply:
yea you are right